package.json

```diff
@@ -17,7 +17,6 @@
     "licenses:list": "license-checker --production --csv --out LICENSE_DEPENDENCIES_ALL",
     "licenses:check": "license-checker --production --failOn \"AGPL-1.0-only; AGPL-1.0-or-later; AGPL-3.0-only; AGPL-3.0-or-later; Beerware; CC-BY-NC-1.0; CC-BY-NC-2.0; CC-BY-NC-2.5; CC-BY-NC-3.0; CC-BY-NC-4.0; CC-BY-NC-ND-1.0; CC-BY-NC-ND-2.0; CC-BY-NC-ND-2.5; CC-BY-NC-ND-3.0; CC-BY-NC-ND-4.0; CC-BY-NC-SA-1.0; CC-BY-NC-SA-2.0; CC-BY-NC-SA-2.5; CC-BY-NC-SA-3.0; CC-BY-NC-SA-4.0; CPAL-1.0; EUPL-1.0; EUPL-1.1; EUPL-1.1;  GPL-1.0-only; GPL-1.0-or-later; GPL-2.0-only;  GPL-2.0-or-later; GPL-3.0; GPL-3.0-only; GPL-3.0-or-later; SISSL;  SISSL-1.2; WTFPL\"",
     "prepublishOnly": "npm run build",
-    "prepare": "husky install",
```

Comment:
Do you also want to remove husky as a dependency?

Reply:
True, actually I'm not sure this specific change is necessary as part of this PR. I'll remove husky in a separate PR.

Reply:
We did agree to remove it as a dependency, didn't we?

Reply:
(though, now looking closer, I realise this does not do that)

Reply:
To be precise this change is due to me misunderstanding where an issue was coming from using `npm link`: it complained about husky, but the problem was in the link target, and not this package

Reply:
Fixed this in 88ab35d, and will properly remove husky in a separate PR